src/components/Pages/ProfilePage/ProfilePage.js

```diff
@@ -283,21 +275,29 @@ class ProfilePage extends React.Component {
                             className=" column col-lg-4 col-md-4 col-md-4 col-sm-4 col-xs-6 card2"
                           >
                             {this.renderCarbonCounterBox()}
-                            {/* <Counter
-                              end={sumOfCarbonScores(this.props.done || [])}
-                              unit={!pref_eq ?? "lbs CO2"}
-                              icon={`fa ${pref_eq?.icon || "fa-leaf"}`}
-                              title={`Number of ${pref_eq?.name}`}
-                              info={
-                                pref_eq?.explanation ||
-                                "Amount your yearly carbon footprint is reduced through the actions you've taken."
-                              }
-                            /> */}
                           </div>
                         </div>
                       </div>
                     </div>
                   </section>
+
+                  {userIsAGuest && (
+                    <div
+                      className="become-valid-from-guest touchable-opacity"
+                      onClick={() =>
+                        this.setState({ wantsToBecomeValidUser: true })
+                      }
+                    >
+                      <p>
+                        You are currently a guest, take these easy steps to
+                        become a valid user
```

Comment:
Let's use "registered user".

Also can you say a sentence or two to educate them on the advantages of becoming one?

Reply:
Can we say "You are You are currently a guest of <community name>, click here to become a member!"?

I'm not sure what word exactly we want to use. "Member" (of a community) is more user friendly than "user" I think.

There is still some disagreement about what guest users should be able to do differently from full users. Personally, I think guest users should only be able to use actions, write testimonials, and RSVP for events. To create or join a team you have to be a full user. If we add further user interaction down the road, like commenting on a testimonial, that kind of thing should only be doable by a full user. Users are automatically more accountable if they're not anonymous.